### PR TITLE
Add instructions for paying asset listing fee

### PR DESCRIPTION
**Pull request by m52go**

### CODE REVIEW

File: exchange/howto/list-asset.adoc [hunk 1]
```diff
@@ -155,6 +155,19 @@ This fee is a Bisq DAO parameter, so it is subject to change, but is 1 BSQ per d
...
+.How to pay the asset listing fee
+--
+There is an asset listing fee section in the DAO section of the software:
+
+.Asset listing fee area
+image::images/asset-listing-fee.png[Asset listing fee area]
+
+Simply select the asset you'd like to re-list, specify a fee to pay according to the rules above, and hit the `Pay fee` button.
+
+You will need BSQ to make the transaction, along with a little BTC for mining fees. If you don't have BSQ, check out <<../../getting-started-dao-traders#acquire-bsq, this doc>> to see how to get some. This is no other way to pay this fee.
```
Comment (devinbileck) [inline exchange/howto/list-asset.adoc]: <s>This</s>**There** is no other way to pay this fee.

Comment (sqrrm) [inline exchange/howto/list-asset.adoc]:
```suggestion
You will need BSQ to make the transaction, along with a little BTC for mining fees. If you don't have BSQ, check out <<../../getting-started-dao-traders#acquire-bsq, this doc>> to see how to get some. There is no other way to pay this fee.
```

Comment (sqrrm) [reply exchange/howto/list-asset.adoc]: The insert suggestion tool on here to make one line fixes is really quite helpful, especially for comments. Good to fix small things that can easily be merged without further testing.

Comment (m52go) [reply exchange/howto/list-asset.adoc]: Very cool...didn't know about this.

**Pull request merged**